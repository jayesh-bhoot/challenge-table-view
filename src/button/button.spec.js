import React from 'react'

import Button from './button'

test('should render a button', () => {
  const wrapper = shallow(<Button> Hello </Button>)
  expect(wrapper).toMatchSnapshot()
})
