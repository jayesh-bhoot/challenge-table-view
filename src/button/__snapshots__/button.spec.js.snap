// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a button 1`] = `
<WithStyles(Button)
  classes={
    Object {
      "root": "WithStyles-Button--root-1",
    }
  }
>
   Hello 
</WithStyles(Button)>
`;
